pkg/caret/tests/testthat/test_models_lfda.R

```diff
@@ -21,4 +23,14 @@ test_that('test lfda model training and prediction', {
   expect_that(dim(tr_dat)[1], equals(dim(transformed.train)[1]))
 })
 
+test_that('test lfda model visualization function', {
+  skip_on_cran()
+  set.seed(1)
+  tr_dat <- twoClassSim(200)
+
+  lfda.model <- lfda(x=tr_dat[,-16],y=tr_dat[,16],r=3)
```

Comment:
I just realized that.  At this point you really should make an `lfda` package and submit it to CRAN.  I'd be happy to help you do that.  If you initialize a github repo, I'll send you a PR to setup testthat/travis/coveralls, etc. and walk you through your first CRAN submission.